node-client/README.md

```diff
@@ -25,8 +25,7 @@ pip install -r requirements.txt
 solution:
 
 ```
-sudo mkdir -p /var/lib/rnode
-sudo chown $USER /var/lib/rnode
+sudo mkdir -p ~/.rnode
```

Comment:
definitely don't need `sudo` or `-p` this.

better question: will it not create the directory automagically now?  

see:
https://github.com/rchain/rchain/blob/64f3c20b13fd3a5ebd4950d0286d2c11b344eafe/rholang/src/main/scala/coop/rchain/rholang/interpreter/Runtime.scala#L92